### PR TITLE
[zh-cn]SYNC labels.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh-cn/docs/concepts/overview/working-with-objects/labels.md [hunk 1]
```diff
@@ -132,10 +132,10 @@ Valid label value:
 * 包含破折号（`-`）、下划线（`_`）、点（`.`）和字母或数字
 
 <!--
-For example, here's the configuration file for a Pod that has two labels
+For example, here's a manifest for a Pod that has two labels
 `environment: production` and `app: nginx`:
 -->
-例如，这是一个有 `environment: production` 和 `app: nginx` 标签的 Pod 配置文件：
+例如，这是具有 `environment: production` 和 `app: nginx` 两个标签的 Pod 配置文件：
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/overview/working-with-objects/labels.md:138]:
```suggestion
例如，以下是一个清单 (manifest)，适用于具有 `environment: production` 和 `app: nginx` 这两个标签的 Pod：
```

**Review by windsonsea — COMMENTED** [review]
Look details and show your knowledge please, thanks.

**Review by Sea-n — COMMENTED** [review]
/approve

**Review by my-git9 — COMMENTED** [review]
/lgtm

**Pull request merged**